File: 3718-smallest-missing-multiple-of-k/3718-smallest-missing-multiple-of-k.cpp
class Solution {
public:
    int missingMultiple(vector<int>& nums, int k) {
        int n=k;
        while(has(nums,k))
        {
            k+=n;
        }
        return k;
    }
    bool has(vector<int>nums,int k)
    {
        for(int i=0;i<nums.size();i++)
        {
            if(nums[i]==k)return true;
        }
        return false;
    }
};

File: 3718-smallest-missing-multiple-of-k/3718-smallest-missing-multiple-of-k.py
class Solution:
    def missingMultiple(self, nums: List[int], k: int) -> int:
        n=k
        while(self.has(nums,k)):
            k+=n
        return k
    def has(self,nums,k):
        for i in nums:
            if i==k:
                return True
        return False
